### ISSUE TITLE
Model changes not saved for a specific model

### ISSUE THREAD

**Issue by aymanhab** [opensim-org/opensim-gui#1266]
https://simtk.org/plugins/phpBB/viewtopicPhpbb.php?f=91&t=12993&p=0&start=0&view=&sid=5894524de1810f276f98b7fc804ad91f

### Actual result

(include screen grabs where appropriate)

**Comment by aymanhab** [comment]
This actually is a bug with the model file missing <objects/> tag under  the tag <MarkerSet>, adding it back everything works as expected. Bad but not common or show stopper. I'd downgrade and close. Maybe email model creator to fix the osim file

**Comment by tkuchida** [comment]
Might be helpful to report deserialization issues to the user as opposed to just ignoring. Related: https://github.com/opensim-org/opensim-core/issues/775

**Comment by aymanhab** [comment]
@tkuchida Absolutely, I was just discussing the scope of what we need to do on the code side rather than documentation/support side.
